demisto_sdk/commands/common/hook_validations/playbook.py

```diff
@@ -348,6 +350,26 @@ def check_script_name(self, pb_script_name, id_set_scripts):
             [pb_script_name == id_set_dict[key].get('name') for id_set_dict in id_set_scripts
              for key in id_set_dict])
 
+    def _is_else_path_in_condition_task(self, task):
+        next_tasks: Dict = task.get('nexttasks', {})
+        return '#default#' in next_tasks
+
+    def verify_condition_tasks_has_else_path(self):  # type: () -> bool
+        """Check whether the playbook conditional tasks has else path
+
+        Return:
+            bool. if the Playbook has else path to all condition task
+        """
+        all_conditions_has_else_path: bool = True
+        tasks: Dict = self.current_file.get('tasks', {})
+        for task in tasks.values():
+            if task.get('type') == 'condition':
+                if not self._is_else_path_in_condition_task(task):
+                    error_message, error_code = Errors.playbook_condition_has_no_else_path(task.get('id'))
+                    if self.handle_error(error_message, error_code, file_path=self.file_path, warning=True):
+                        self.is_valid = all_conditions_has_else_path = False
```

Comment:
in case of multiple tasks, I think we should still output one error line and just list all task ids.